src/votekit/elections/election_types/ranking/top_two.py

```diff
@@ -0,0 +1,80 @@
+from .abstract_ranking import RankingElection
+from ....pref_profile import PreferenceProfile
+from ...election_state import ElectionState
+from ....utils import first_place_votes, remove_cand
+from ..ranking import Plurality
+from typing import Optional
+
+
+class TopTwo(RankingElection):
+    """
+    Eliminates all but the top two plurality vote-getters, and then conducts a runoff between them,
+    reallocating other ballots.
+
+    Args:
+        profile (PreferenceProfile): Profile to conduct election on.
+        tiebreak (str, optional): Tiebreak method to use. Options are None, 'random', and 'borda'.
+            Defaults to None, in which case a tie raises a ValueError.
+
+    """
+
+    def __init__(self, profile: PreferenceProfile, tiebreak: Optional[str] = None):
+        self.tiebreak = tiebreak
+        super().__init__(profile, score_function=first_place_votes, sort_high_low=True)
+
+    def _is_finished(self):
+        # two round election
+        if len(self.election_states) == 3:
+            return True
+        return False
+
+    def _run_step(
+        self, profile: PreferenceProfile, prev_state: ElectionState, store_states=False
+    ) -> PreferenceProfile:
+        """
+        Run one step of an election from the given profile and previous state.
+
```

Comment:
Put the description of the algo here

Reply:
In the description, it would be a good idea to remind people that the votes are reallocated using the score function of the parent class